TADashboard/src/test/java/tests/general_settings/NegativeDPGeneralSettingsTest.java

```diff
@@ -46,6 +51,26 @@ public void tc069_UserIsUnableToProceedToNextStepOrFinishCreatingDataProfileIfNa
         Assert.assertEquals(DriverUtils.getAlertText(), "Please input profile name.", "Dialog doesn't display correct message");
     }
 
+    @Test(description = "Verify that all data profile types are listed in priority order under 'Item Type' dropped down menu")
+    public void tc073_AllDataProfileTypesAreListed() {
+            List<String> optionInTypeItem = new ArrayList<>();
+            optionInTypeItem.add("test modules");
+            optionInTypeItem.add("test cases");
+            optionInTypeItem.add("test objectives");
+            optionInTypeItem.add("data sets");
+            optionInTypeItem.add("actions");
+            optionInTypeItem.add("interface entities");
+            optionInTypeItem.add("test results");
+            optionInTypeItem.add("test case results");
+            optionInTypeItem.add("test suites");
+            optionInTypeItem.add("bugs");
+            Logger.verify("Items are listed in priority order");
+            dataProfilesGeneralSettingsPage.clickItemTypeComboBox();
+            for (String option : optionInTypeItem) {
+                Assert.assertTrue(dataProfilesGeneralSettingsPage.isOptionVisible(option),
```

Comment:
How do you check they are listed in priority order?

If it has `actions` before `data sets`. Is this test failed?

Reply:
I check based expected result in the test case. And if  `action` before `data set`, actual result will unmatch with expected result so this test failed